.github/workflows/build-images.yml

```diff
@@ -51,14 +51,15 @@ jobs:
         env:
           APP_ID: ${{ matrix.environment.app_id }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           DEPLOY_AWS_ACCESS_KEY_ID: ${{ secrets[matrix.environment.aws_deploy_key] }}
           DEPLOY_AWS_SECRET_ACCESS_KEY: ${{ secrets[matrix.environment.aws_deploy_secret] }}
-          AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
-          SENTRY_DSN: ${{ secrets.SENTRY_TOKEN }}
-          PERM_ENV: ${{ matrix.environment.perm_env }}
+          NEW_RELIC_LICENSE_KEY: ${{ secrets.NEW_RELIC_LICENSE_KEY }}
```

Comment:
This looks good!

You'll also need to set the `NEW_RELIC_APPLICATION_NAME` variable, in the same way as we're doing `APP_ID`.

Reply:
And it needs to be piped all the way through, just like the license key!

Reply:
aah ok.. will do that..